### DISCUSSION TITLE
Update to latest net core

### DISCUSSION THREAD

**Discussion by AnFa1978** [actions/virtual-environments#2671] [Installed software]
Hi @ All,

does anybody know schedule until update to .NET 3.1.406 and 5.0.103 is planed? Thanks in advance.

Kind regards,
Andreas

**Comment by AlenaSviridenko** [comment]
Hi @AnFa1978,
these versions will be available on the images with the next deployment (approx. next week). They were released only yesterday and we will be able to include them to next rollout.
Meanwhile, consider using [actions/setup-dotnet](https://github.com/actions/setup-dotnet) for GitHub Action or [Use .NET Core task](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/dotnet-core-tool-installer?view=azure-devops) for AzDO to install any version you'd like in runtime.
